fleet_adapter_ecobot/EcobotClientAPI.py

```diff
@@ -254,22 +254,23 @@ def stop(self):
         return False
 
     def current_map(self):
-        url = self.prefix + f"/gs-robot/real_time_data/robot_status"
+        url = self.prefix + f"/gs-robot/data/current_map"
```

Comment:
Is this endpoint change really valid? I don't see any reference to it in the Gaussian API manual. 
http://download.gs-robot.com/gs_api/api.html#1

The `ecobot_sim_server.py` is an emulator of the Gaussian Ecobot server written by us to mimic the response form the actual gaussian robots. So any changes made to EcobotClientAPI or ecobot_sim_server need to be backed by suitable changes in the official Gaussian API

Reply:
Thanks , that was my question, because I'm not familiar with the EcobotAPI. The page is in Chinese, I don't know if there is an English version. But I imagine that chning the calls in the API is not the bets practice. I'll see to revert that call and change form the simulated server side. Is that better as solution?
Becuase its difficult form e to read the chinise docs

Reply:
Is there a reason for the _state() method to be the in the EcobotClientAPI.py? Because it seems its not used anywhere, but I'm probably wrong. Is to know if its cleanable or not.

Reply:
> The page is in Chinese, I don't know if there is an English version

I'm able to use Page Translate to translate to English.

Yeah changing things on the server side is recommended especially if the current response from the server is incorrect.

Reply:
Reverted the APIClient changes and updated the server accordingly. Please tell us if there si something else to be corrected :)